internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,17 @@
+begin;
+
+create table server_type_enm (
+  name text primary key not null,
+  constraint only_predefined_server_types_allowed
+    check (name = ANY (array['controller'::text, 'worker'::text]))
```

Comment:
We have a [standard](https://github.com/hashicorp/boundary/blob/c9e4c877b89038c9e9d4f1edecb15a1123073519/internal/db/schema/migrations/postgres/2/03_oidc_enm.up.sql#L8-L10) [pattern](https://github.com/hashicorp/boundary/blob/c9e4c877b89038c9e9d4f1edecb15a1123073519/internal/db/schema/migrations/postgres/10/03_credential.up.sql#L243-L249) we use for checking for valid values. Is there a specific reason for not following that pattern?

Reply:
Addressed [above](https://github.com/hashicorp/boundary/pull/1386#discussion_r666504684).